extra-dev/packages/bonak/bonak.dev/opam

```diff
@@ -1,21 +1,21 @@
 opam-version: "2.0"
-name: "bonak"
-version: "master"
-synopsis: "An indexed construction of truncated n-types"
-maintainer: "Ramkumar Ramachandra <r@artagnon.com>"
-authors: [ "Ramkumar Ramachandra <r@artagnon.com>"
-           "Hugo Herbelin <hugo.herbelin@inria.fr>" ]
-license: "MIT"
-homepage: "https://github.com/artagnon/bonak"
-bug-reports: "https://github.com/artagnon/bonak/issues"
-dev-repo: "git+https://github.com/artagnon/bonak"
-depends: [
-  "coq"  { >= "8.15.1" & < "8.16" }
-  "dune" { >= "2.9.1" }
-]
-build: [
-  ["dune" "build"]
-]
-url {
-  src: "git+https://github.com/artagnon/bonak.git#master"
-}
+ name: "bonak"
+ version: "master"
```

Comment:
```suggestion
```
The version is already defined by the path name `bonak.dev`.

Also, please do not insert additional indentation compared to before, this makes it harder to process package definitions by tools.